frontend/src/components/Step4.module.css

```diff
@@ -0,0 +1,174 @@
+.page4Form {
+  background-color: white;
+  box-shadow: 0 5px 10px rgba(0, 0, 0, 0.25);
+}
+
+.formRow {
+  display: flex;
+  justify-content: space-between;
+  margin-left: 100px;
+  margin-right: 130px;
+  padding-bottom: 30px;
+}
+
+@media (max-width: 1250px) {
+  .formRow {
+    display: flex;
+    flex-direction: column;
```

Comment:
![image](https://github.com/TritonSE/CCIDC-Application-Overhaul/assets/59126064/e3f3cb8a-96d7-4531-b865-3df2bb0e6840)
Ideally there should be padding when the fields flex. See personal Inforomation form. We want to keep the styling accross the steps as consistent as possible